ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js

```diff
@@ -91,15 +91,20 @@ const EditGasItem = ({ priorityLevel, onClose }) => {
   };
 
   return (
-    <div
+    <button
       className={classNames('edit-gas-item', {
         'edit-gas-item-selected': priorityLevel === estimateUsed,
         'edit-gas-item-disabled':
           priorityLevel === PRIORITY_LEVELS.DAPP_SUGGESTED &&
           !dappSuggestedGasFees,
       })}
-      role="button"
+      disabled={
+        priorityLevel === PRIORITY_LEVELS.DAPP_SUGGESTED &&
+        !dappSuggestedGasFees
```

Comment:
[nit] these lines are repeated on 98/99 -> could be extracted in to 

```
const disabled = priorityLevel === PRIORITY_LEVELS.DAPP_SUGGESTED &&
        !dappSuggestedGasFees
```

Reply:
it looks ok to me actually